### PR TITLE
CentOS 8

### PR DESCRIPTION
**Pull request by enbrnz**
- Molecule testing
- Package installation handled between CentOS / RHEL 7 & 8 with first_found

### CODE REVIEW

**Comment by enbrnz** [comment]
PHP version handling has been removed, if you want to install another PHP version other than the default PHP 7.2 change the app stream like so (before applying the role):
https://docs.centos.org/en-US/8-docs/managing-userspace-components/assembly_managing-versions-of-appstream-content/
`dnf module enable php:7.4` (as an example for PHP 7.4)

**Comment by enbrnz** [comment]
- `make` is required for pecl modules

File: .drone.yml [hunk 1]
```diff
@@ -207,11 +243,8 @@ depends_on:
 - linting
 - testing-ubuntu1804
 - testing-centos7
+- testing-centos8
 - testing-opensuse15
 - documentation
 
----
```
Comment (xoxys) [inline .drone.yml:213]: Is the drone config signed?

**Review by xoxys — APPROVED** [review]
LGTM

**Pull request merged**